### PR TITLE
Added application file

### PR DESCRIPTION
**Pull request by Allenkeys**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello");
```
Comment (chuka00) [inline app/app.js:1]: needs more design pattern

Comment (IsraelChidera) [inline app/app.js:1]: you should have written Console.Writeline()

**Review by chuka00 — CHANGES_REQUESTED** [review]

**Review by IsraelChidera — COMMENTED** [review]
Add more code

File: app/app.js [hunk 2]
```diff
@@ -1 +1,4 @@
-console.log("Hello");
\ No newline at end of file
+function display(){
+  console.log("Hello");
+}
```
Comment (IsraelChidera) [inline app/app.js:3]: Perfect!

File: app/app.js [hunk 3]
```diff
@@ -1 +1,4 @@
...
+display();
```
Comment (chuka00) [inline app/app.js:4]: This is better

**Review by chuka00 — APPROVED** [review]

**Review by IsraelChidera — APPROVED** [review]
Good 

@chuka00 check it out

**Pull request merged**